packages/scaffold-ui/src/partials/w3m-social-login-widget/index.ts

```diff
@@ -58,34 +59,37 @@ export class W3mSocialLoginWidget extends LitElement {
 
   // -- Render -------------------------------------------- //
   public override render() {
-    const socials = this.features?.socials
-
-    if (!this.authConnector || !socials || !socials?.length) {
-      return null
-    }
-
     return html`
       <wui-flex
         class="container"
         flexDirection="column"
         gap="xs"
-        .padding=${['0', '0', 'xs', '0'] as const}
         data-testid="w3m-social-login-widget"
       >
         ${this.topViewTemplate()}${this.bottomViewTemplate()}
       </wui-flex>
-      ${this.separatorTemplate()}
     `
   }
 
   // -- Private ------------------------------------------- //
   private topViewTemplate() {
-    const socials = this.features?.socials
+    const isCreateWalletPage = this.walletGuide === 'explore'
```

Comment:
I like starting boolean names with `is`, `has` and `should`, but i don't see it being used a lot in the codebase. Shall we maybe start doing that ?